lib/ui/widgets/album_tile.dart

```diff
@@ -13,9 +12,14 @@ class AlbumTile extends StatelessWidget {
     return Stack(
       fit: StackFit.expand,
       children: [
-        CachedImage(
```

Comment:
We need to keep caching though

Reply:
Whoops, this is why you dont copy paste lol, such a small PR...